### PR TITLE
Feat/seed data

### LINKED ISSUE #57: Extend settings
Choose which metrics other than cpu usage that should be taken into account 

    Choose how often to train 

    View hyperparameters 

    Optuna configuration

### PR DESCRIPTION
**Pull request by Jaffenheimer**
This resolves #57 and #80

### CODE REVIEW

File: Autoscaler.DbUp/Scripts/0006-SeedData.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+-- Insert sample data into Services table
+INSERT INTO autoscaler.Services (Id, Name)
+VALUES ('1a2b3c4d-1111-2222-3333-444455556666', 'Image Recognition'),
+       ('2b3c4d5e-1111-2222-3333-444455556666', 'Speech-to-Text'),
+       ('3c4d5e6f-1111-2222-3333-444455556666', 'Recommendation System');
+
+-- Insert sample data into Settings table
+INSERT INTO autoscaler.Settings (Id, ServiceId, ScaleUp, ScaleDown, ScalePeriod, TrainInterval, ModelHyperParams, OptunaConfig)
+VALUES ('a1b2c3d4-aaaa-bbbb-cccc-ddddeeeeffff', '1a2b3c4d-1111-2222-3333-444455556666', 5, 2, 10, 30,
+        '{"learning_rate": 0.01, "batch_size": 32, "epochs": 50}',
```
Comment (Mast3rwaf1z) [inline Autoscaler.DbUp/Scripts/0006-SeedData.sql:12]: this is pretty cool, good idea

**Review by Mast3rwaf1z — APPROVED** [review]
looks good, i would like tests, but i will create an issue looking into it later
